packages/onboarding/src/cart/index.tsx

```diff
@@ -0,0 +1,267 @@
+/* eslint-disable no-restricted-imports */
```

Comment:
Is this a good idea? Calypso [has a dependency](https://github.com/Automattic/wp-calypso/blob/3511f6b53c787a8c5b8a2ed15f8ddbc575ce3d82/client/package.json#L49) on the `onboarding` package, and by doing this, the `onboarding` package will have a dependency on calypso, creating a circular dependency. I don't know a ton about bundling, but I'm pretty sure that's a bad thing.

It looks like you're adding this so that you can use the `cartManagerClient` from calypso, but you can recreate that using the `shopping-cart` package here if you like, or inject it as a dependency wherever this is used.

Reply:
Thank you @sirbrillig, I will take this into consideration and try to remove the circular dependency.